README.md

```diff
@@ -26,7 +26,7 @@ worldengine ancient_map -w seed1.world
 ![](https://raw.githubusercontent.com/Mindwerks/worldengine-data/master/images/examples/ancient_map_seed1.png)
 
 
-We started a [google group](https://groups.google.com/forum/?hl=en#!forum/worldengine): if you have ideas, problems, suggestions or want to contribute please [join us](https://groups.google.com/forum/?hl=en#!forum/worldengine)!
+We started a [Google group](https://groups.google.com/forum/?hl=en#!forum/worldengine): Should you have ideas, problems, suggestions or want to contribute, please [join us](https://groups.google.com/forum/?hl=en#!forum/worldengine)!
```

Comment:
I like it that way, although the topic seems to be a bit fuzzy.
http://www.apastyle.org/learn/faqs/colon-use.aspx
> If the clause following the colon is a complete sentence, it begins with a capital letter.

or
> The NYPL Writer's Guide urges consistency within a document; the Chicago Manual of Style says you may begin an
> independent clause with a lowercase letter unless it's one of those two things (a quotation or more than one sentence).

So...both ways are possible, it seems.